dGame/dUtilities/VanityUtilities.cpp

```diff
@@ -253,23 +237,27 @@ void VanityUtilities::ParseXML(const std::string& file) {
 				for (auto* key = configElement->FirstChildElement("key"); key != nullptr;
 					key = key->NextSiblingElement("key")) {
 					// Get the config data
-					auto* data = key->Attribute("data");
+					auto* data = key->GetText();
 					if (!data) continue;
 
 					LDFBaseData* configData = LDFBaseData::DataFromString(data);
+					if (configData->GetKey() == u"useLocationsAsRandomSpawnPoint" && configData->GetValueType() == eLDFType::LDF_TYPE_BOOLEAN){
+						useLocationsAsRandomSpawnPoint = static_cast<bool>(configData);
+						continue;
+					}
 					keys.push_back(configData->GetKey());
 					config.push_back(configData);
 				}
 			}
 			if (!keys.empty()) config.push_back(new LDFData<std::vector<std::u16string>>(u"syncLDF", keys));
 
-			VanityObject objectData;
-			objectData.m_Name = name;
-			objectData.m_LOT = std::stoi(lot);
-			objectData.m_Equipment = inventory;
-			objectData.m_Phrases = phraseList;
-			objectData.m_Script = scriptName;
-			objectData.m_Config = config;
+			VanityObject objectData {
+				.m_Name = name,
+				.m_LOT = std::stoi(lot),
```

Comment:
try parse here please

Reply:
Ah yeah, crap. Forgot to replace all of those